### PR TITLE
4432 bug configuration from dataset

### LINKED ISSUE #4456: [BUG]: Model configuration from document GoLLM extraction isn't updating distributions
There is an issue with the GoLLM model config from document job where we will always receive a distribution object that is the same as the original that was given to the prompt.  The way we create configurations right now is that if a `distribution` is present the distribution value will be applied instead of the value.  We should be given either value or distribution not both.

Here is an example.  Theses are 2 json objects that were received from the same GoLLM job.  We can see that the `value` of β and γ change between the 2, but the distribution given back remains the same as the one that was given to the prompt.

Side note: Initials don't seem to ever be considered from the model config for document job.

```json
{
   "name":"China",
   "description":"Data from January to June 2020, peak around 16 February 2020",
   "parameters":[
      {
         "id":"β",
         "value":0.4,
         "name":"β",
         "description":"Infection rate",
         "units":{
            "expression":"1/(person*day)",
            "expression_mathml":"<apply><divide/><cn>1</cn><apply><times/><ci>person</ci><ci>day</ci></apply></apply>"
         },
         "distribution":{
            "type":"StandardUniform1",
            "parameters":{
               "minimum":0.4,
               "maximum":0.8
            }
         }
      },
      {
         "id":"γ",
         "value":0.1,
         "name":"γ",
         "description":"Recovery rate",
         "units":{
            "expression":"1/day",
            "expression_mathml":"<apply><divide/><cn>1</cn><ci>day</ci></apply>"
         },
         "distribution":{
            "type":"StandardUniform1",
            "parameters":{
               "minimum":0.1,
               "maximum":0.3
            }
         }
      }
   ]
}
```

```json
{
   "name":"South Korea",
   "description":"Data from February to June 2020, effective control measures",
   "parameters":[
      {
         "id":"β",
         "value":0.3,
         "name":"β",
         "description":"Infection rate",
         "units":{
            "expression":"1/(person*day)",
            "expression_mathml":"<apply><divide/><cn>1</cn><apply><times/><ci>person</ci><ci>day</ci></apply></apply>"
         },
         "distribution":{
            "type":"StandardUniform1",
            "parameters":{
               "minimum":0.4,
               "maximum":0.8
            }
         }
      },
      {
         "id":"γ",
         "value":0.05,
         "name":"γ",
         "description":"Recovery rate",
         "units":{
            "expression":"1/day",
            "expression_mathml":"<apply><divide/><cn>1</cn><ci>day</ci></apply>"
         },
         "distribution":{
            "type":"StandardUniform1",
            "parameters":{
               "minimum":0.1,
               "maximum":0.3
            }
         }
      }
   ]
}
```

### PR DESCRIPTION
**Pull request by dgauldie**
# Description

* Include a summary of the changes and the related issue. 
* Include relevant motivation and context.

Resolves #4456, #4546, #4545

### CODE REVIEW

File: packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-drilldown.vue [hunk 1]
```diff
@@ -411,25 +411,31 @@ const extractConfigurationsFromInputs = async () => {
...
+
 	if (documentId.value) {
 		const resp = await configureModelFromDocument(
 			documentId.value,
 			model.value.id,
 			props.node.workflowId,
 			props.node.id
 		);
-		state.documentModelConfigTaskId = resp.id;
+		state.modelConfigTaskIds.push(resp.id);
```
Comment (Tom-Szendrey) [inline packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-drilldown.vue:422]: todo multiple documents?

**Review by YohannParis — APPROVED** [review]

File: packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/gollm/GoLLMController.java [hunk 2]
```diff
@@ -330,77 +331,66 @@ public ResponseEntity<TaskResponse> createConfigFromDatasetTask(
...
 		}
 
-		final ConfigureFromDatasetResponseHandler.Input input = new ConfigureFromDatasetResponseHandler.Input();
-		input.setDatasets(datasets);
+		final ConfigureModelFromDatasetResponseHandler.Input input = new ConfigureModelFromDatasetResponseHandler.Input();
+		input.setDataset(dataArray);
 		// stripping the metadata from the model before its sent since it can cause
 		// gollm to fail with massive inputs
 		model.get().setMetadata(null);
 		input.setAmr(model.get().serializeWithoutTerariumFields());
```
Comment (kbirk) [inline packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/gollm/GoLLMController.java:374]: Do we need to use `serializeWithoutTerariumFieldsKeepId` here to preserve the models id?

**Review by kbirk — APPROVED** [review]

File: packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-drilldown.vue (hunk 1 above)
Comment (dgauldie) [reply packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-drilldown.vue]: fixed

File: packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/gollm/GoLLMController.java (hunk 2 above)
Comment (dgauldie) [reply packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/gollm/GoLLMController.java:374]: nope, I now populate with the nil uuid if not found. We then postprocess and add the model id back in. caching can remain

**Review by Tom-Szendrey — APPROVED** [review]
Sweet!

**Pull request merged**